aea/crypto/plugin.py

```diff
@@ -129,7 +129,8 @@ def _get_plugins(group: str) -> List[Plugin]:
     :param group: the plugin group.
     :return: a mapping from plugin name to Plugin objects.
     """
-    entry_points: List[EntryPoint] = list(iter_entry_points(group=group))
+
+    entry_points: List[EntryPoint] = list(WorkingSet().iter_entry_points(group=group))
```

Comment:
what is the difference?

Reply:
`WorkingSet()` so looks like default workingset populated once, and iter_entry_points is the same during python process lifetime, even you added removed python packages.